python/sdist/amici/de_export.py

```diff
@@ -1992,6 +1992,9 @@ def _compute_equation(self, name: str) -> None:
                 for ie in range(self.num_events())
             ]
 
+        elif name == "deltasx" and self.num_states_solver() * self.num_par() == 0:
+            self._eqs[name] = []
+
         elif name == "deltasx":
```

Comment:
wouldn't it be better to integrate with the code below, i.e. add a special case for `self.num_states_solver() * self.num_par() == 0`?

Reply:
Generally, yes. But I didn't want increase the nesting depth further. It's already a mess.